### DISCUSSION TITLE
Can I open and edit the scenes from procthor-10k dataset in Unity.editor?

### DISCUSSION THREAD

**Discussion by MingZwhy** [allenai/ai2thor#1127] [Q&A]
Now I can open and edit the scenes from ai2thor in Unity.editor.
But for scenes from procthor-10k, I can only experence them in colab using python API, is there any way to open them in Unity.editor?

**Comment by Young-Loser** [comment]
May I ask how you opened IoA in Unity.editor and edited scenes in AI2Thor? Can you explain it in detail? Thank you very much!

**Comment by Young-Loser** [comment]
How did you open and edit scenes in ai2hor in Unity.editor? Can you explain it in detail? Thank you very much!

**Reply by MingZwhy** [reply]
After you git clone the repo, you can find a folder named "unity".
Then you should download the Unity Hub and donwload a unity.editor in the Unity Hub.
After Unity Hub and unity.editor are all set, you can open folder "unity" in Unity Hub.
In the unity.editor, you can see a subfolder named "scence" which includes all the rooms in ai2thor. You can open and edit any room(xxx.scene) in this folder in unity editor.

**Comment by zjwzcx** [comment]
Same question.

**Reply by zjwzcx** [reply]
Solved, referring to: https://github.com/allenai/ai2thor/issues/1128
